config.json

```diff
@@ -41,6 +41,17 @@
         "text_formatting"
       ]
     },
+    {
+      "slug": "resistor-color-duo",
+      "uuid": "98ae146a-67f9-49ae-9cd6-00882a374248",
+      "core": false,
+      "unlocked_by": "two-fer",
```

Comment:
I would prefer if this was unlocked after resistor-color. Was there a particular reason you have it unlocked after two-fer?

Reply:
No, was just thinking this would be good to unlock one step down the track from `resistor-color`. I'm fine with making this change. :)

Reply:
Updated, this is now unlocked by `resistor-color`.

```
core
----
├─ hello-world [1]
│  ├─ resistor-color [2]
│  │  └─ resistor-color-duo [2]
```

Reply:
Seems like CI is failing,

```
The exercise 'resistor-color-duo' is being unlocked by a non-core exercise. Non-core exercises can only be unlocked by core exercises.
```

Reply:
@amscotti Ah okay, it can it be unlocked after two-fer.

Reply:
@Stargator Updated! :)

Reply:
Sorry for the back and forth

Reply:
It's all good, no issue. 😄